buildSrc/src/main/kotlin/non-multiplatform.publish-conventions.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+//     Regular java modules need 'java-library' plugin for proper publication
+    `java-library`
+    id("publish-conventions")
+}
+
+// MPP projects pack their sources automatically, java libraries need to explicitly pack them
+val sourcesJar by tasks.registering(Jar::class) {
+    archiveClassifier.set("sources")
+    from("src/main/kotlin")
```

Comment:
If this convention assumes there's a `src/main/kotlin` directory shouldn't it apply the Kotlin/JVM plugin? 🤔